sdks/python/apache_beam/runners/direct/transform_evaluator.py

```diff
@@ -436,8 +436,9 @@ def finish_bundle(self):
       bundles = [bundle]
     else:
       bundles = []
-    input_pvalue = self._applied_ptransform.inputs
-    if not input_pvalue:
+    if self._applied_ptransform.inputs:
```

Comment:
Should we separate this fix to another PR? We can get cherry pick the bug fix to the release branch.